app/Http/Controllers/PostsController.php

```diff
@@ -9,7 +9,15 @@ class PostsController extends Controller
 {
     public function index(Request $request)
     {
-        $posts = Post::published()->orderBy('published_at', 'desc')->paginate(10);
+        $search = $request->input('search');
+
+        $posts = null;
+        if (empty($search)) {
+            $posts = Post::published()->orderBy('published_at', 'desc')->paginate(10);
+        } else {
+            $posts = Post::search($search)->orderBy('published_at', 'desc')->paginate(10);
```

Comment:
```suggestion
            $posts = Post::search($search)->paginate(10);
```
just a nit :D 
but keep it sorted by relevance